spec.md

```diff
@@ -41,7 +40,7 @@ behind the specification, see the [CloudEvents Primer](primer.md) document.
 
 ## Notations and Terminology
 
-### Notational Conventions
+## Notational Conventions
```

Comment:
I think 3 hashes are correct - this is meant to be under "Notations and Terminology"